layouts/shortcodes/blocks/case-studies.html

```diff
@@ -13,7 +13,7 @@ <h3>{{ $caseStudiesSection.Title }}</h3>
 				{{ $logo = .Resources.GetMatch "**logo*.png" }}
 			{{ end }}
       <div>
-        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="{{ .Title }}">{{ end }}
+        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="">{{ end }}
```

Comment:
I believe this alt attributes are there to improve accessibility.

Reply:
ok